app/Module.java

```diff
@@ -90,17 +87,6 @@ protected void configure() {
                 .toProvider(EmailSenderProvider.class)
                 .in(Singleton.class);
 
-        // Bindings for all user context related
-        bind(Locale.class)
-                .toProvider(LocaleFromUrlProvider.class)
-                .in(RequestScoped.class);
-        bind(CountryCode.class)
-                .toProvider(CountryFromSessionProvider.class)
-                .in(RequestScoped.class);
-        bind(CurrencyUnit.class)
-                .toProvider(CurrencyFromCountryProvider.class)
-                .in(RequestScoped.class);
-
```

Comment:
Removed from Module.java, because this file belongs to Starter Project and is therefore never updated with new releases, what makes migration more difficult. Now moved to LocalizationModule, which is enabled by default on reference.conf (default configuration), and can be disabled via configuration as explained in #671 in order to bind your own implementations.